server/routes/popVerifyRoutes.ts

```diff
@@ -0,0 +1,128 @@
+import { type RequestHandler, Router } from 'express'
+import path from 'path'
+import asyncMiddleware from '../middleware/asyncMiddleware'
+import setUpMultipartFormDataParsing from '../middleware/setUpMultipartFormDataParsing'
+
+export default function routes(): Router {
+  const router = Router()
+
+  const get = (routePath: string | string[], handler: RequestHandler) => router.get(routePath, asyncMiddleware(handler))
+  const post = (routePath: string | string[], handler: RequestHandler) =>
+    router.post(routePath, asyncMiddleware(handler))
+  router.use(setUpMultipartFormDataParsing())
+
+  get('/', async (req, res, next) => {
```

Comment:
We need to land here from the end of the one login registration journey, instead of going straight into pop welcome page

Reply:
redirect amended